home/ivp-planning/getting-started/importing-pdfs.md

```diff
@@ -2,30 +2,28 @@
 
 In iVP it is possible to import PDFs that will act as an overlay for the [floor plan](../user-interface/the-floor-plan.md). This can be useful for illustration or when it is necessary to transform a 2D (factory) plan into a 3D scene.
 
+## Importing PDFs step by step:
 
-{% embed url="https://youtu.be/MuLt94b64O8?t=203" %}
-How to import and export PDFs (german)
-{% endembed %}
+To import a PDF, click on **File -> Import**. A new window will open that allows you to navigate to the folder containing the PDF you want to import.
 
-## Importing PDFs step by step:
+![](../../../.gitbook/assets/pdf_import_pdf_menu_entry.jpg)
+
+As soon as you have found the file, simply double-click on it or select it and click **Open**.
+
+![](../../../.gitbook/assets/pdf_import_pdf_file_dialogue.jpg)
 
-To import a PDF click on **File -> PDF -> Import PDF**. A new window will open that allows you to navigate to the folder that contains the PDF you want to import.
+Before iVP actually processes the PDF and imports it, you will be asked to set some parameters that specify how it will be loaded. The options are:
 
-![](../../../.gitbook/assets/iVP\_pdf\_import\_pfd\_menu\_entry.jpg)
+* **Page Size**: Changes the size of the imported PDF, you can either choose from a preset or manually customize the size of the file.
 
-As soon as you found the file, simply double click or it or select it and click **Open**.
+* **Width Original File**: Displays the width of the original file.
 
-![](../../../.gitbook/assets/iVP\_pdf\_import\_pdf\_file\_dialogue.jpg)
+* **Scale**: Adjusts the scale of the imported file.
 
-Before iVP actually processes PDF and imports it, you will be asked to set some parameters that specify it will be loaded. The options are:
+* **Resolution**: Adjusts the resolution of the imported file.
 
-* _**PPI:**_** t**his value represents the resolution of the PDF to import (most PDFs have a default resolution of 72 PPI so it is usually not necessary to change this value)
-* _**Scale:**_ this value represents the scale of the PDF to import, e.g. 1:100 or 1:50
-* _**Resize floor to PDF scale:**_ if checked the size of the floor plan will be adjusted to be equal to the dimensions of the imported PDF
-* _**Transparent background:** i_f checked the usually white background of the imported PDF will be removed so that only its contents are visible
+* **Name**: Displays and allows you to change the name of the imported file.
 
-{% hint style="info" %}
-PDF PPI and PDF scale are interdependent values. Therefore the PPI value can also be used to compensate an incorrect PDF scale. However, it is usually not necessary to change the PPI value.
-{% endhint %}
+* **Add to Library**: Adds and saves the imported item to your [personal library](../user-interface/library-panel.md).
```

Comment:
```suggestion
* **Add to Library**: Saves the imported item to your [personal library](../user-interface/library-panel.md).
```

Reply:
I wrote "add and save" so it'd be clearer that the imported item can also be used for other projects.

Reply:
I think just "saved" is enough. The rest is also explained in the linked page.

Reply:
Okay, will make the change to "saved".